doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -127,16 +127,37 @@ analysis on inductive or co-inductive objects (see :ref:`variants`).
       | @one_term_with_bindings
 
    Performs case analysis by generating a subgoal for each constructor of the
-   inductive or co-inductive type in the :n:`@one_term` in :n:`@induction_arg`.
-   :n:`@one_term` (or the conclusion of its type--example
-   :ref:`here<example_destruct_ind_concl>`) must be of inductive or co-inductive type.
-   In each new subgoal, the tactic replaces :n:`@one_term` with the associated
-   constructor (applied to its arguments).   Unlike :tacn:`induction`,
+   inductive or co-inductive type selected by :n:`@induction_arg`.  The selected
+   subterm, after possibly doing an :tacn:`intros`, must have
+   an inductive or co-inductive type.  Unlike :tacn:`induction`,
    :tacn:`destruct` generates no induction hypothesis.
 
-   The :n:`@one_term` may contain holes that are denoted by “_”. In this case,
-   the tactic selects the first subterm that matches the pattern and performs
-   case analysis using that subterm.
+   In each new subgoal, the tactic replaces the selected subterm with the associated
+   constructor (applied to its arguments).
+
+   + If :n:`@one_term` (in :n:`@induction_arg`)
+     is an identifier :n:`@ident` denoting a quantified variable of the
+     goal, then :n:`destruct @ident` behaves like :tacn:`intros` :n:`until
+     @ident; destruct @ident`. :n:`@one_term` must have an inductive type.
```

Comment:
This paragraph is a bit mixed up between the specific `ident` case and the generic `one_term` case. Since this last sentence applies in all cases, it could be factorized somewhere.